hummingbot/connector/exchange/bittrex/bittrex_exchange.pyx

```diff
@@ -709,7 +709,7 @@ cdef class BittrexExchange(ExchangeBase):
                 "type": "LIMIT",
                 "quantity": f"{amount:f}",
                 "limit": f"{price:f}",
-                "timeInForce": "POST_ONLY_GOOD_TIL_CANCELLED"
+                "timeInForce": "POST_ONLY_GOOD_TIL_CANCELED"
```

Comment:
This can't be changed. The correct value according to Bittrex API documentation is `POST_ONLY_GOOD_TIL_CANCELLED`